EGO MAUL Enhancement.user.js

```diff
@@ -0,0 +1,265 @@
+// ==UserScript==
+// @name         EdgeGamers MAUL Enhancement
+// @namespace    https://github.com/blankdvth/eGOScripts
+// @version      2.0.0
+// @description  Add various enhancements & QOL additions to the EdgeGamers MAUL page that are beneficial for CS Leadership members.
+// @author       blank_dvth, Left, Skle, MSWS
+// @match        https://maul.edgegamers.com/*
+// @icon         https://www.google.com/s2/favicons?sz=64&domain=edgegamers.com
+// @require      https://peterolson.github.io/BigInteger.js/BigInteger.min.js
+// @require      https://raw.githubusercontent.com/12pt/steamid-converter/master/js/converter-min.js
+// @resource    admins https://gist.githubusercontent.com/MSWS/310f334dbaec82bfc30efb174eb311fc/raw/19c693d9e3b37ffc066a8d25e88cc08930778f99/admins.txt
+// @grant       GM_getResourceText
+// ==/UserScript==
+
+'use strict';
+let knownAdmins = {}; // Known admin list
+
+/**
+ * Adds a preset button to the div
+ * @param {string} name Name of button
+ * @param {HTMLDivElement} div Div to add to
+ * @param {function(HTMLElementEventMap)} func Function to call on click
+ */
+function addPreset(name, div, func) {
+    div.appendChild(createPresetButton(name, func));
+}
+
+/**
+ * Creates a preset div
+ * @returns {HTMLDivElement} Div to add presets to
+ */
+function createPresetDiv() {
+    var div = document.createElement("div");
+    var subtitle = document.createElement("h4");
+    var child_container = document.getElementById("child_container")
+    div.id = "preset_div";
+    div.style.display = "flex";
+    div.style.flexDirection = "row";
+    div.style.paddingLeft = "15px";
+    div.style.paddingBottom = "10px";
+    subtitle.innerHTML = "Presets";
+    subtitle.style.paddingLeft = "15px";
+    child_container.insertBefore(div, document.querySelector("form"));
+    child_container.insertBefore(subtitle, div)
+    return div;
+}
+
+/**
+ * Creates a preset button
+ * @param {string} text Button text
+ * @param {function(HTMLElementEventMap)} callback Function to call on click  
+ * @returns {HTMLButtonElement} Button
+ */
+function createPresetButton(text, callback) {
+    var button = document.createElement("button");
+    button.classList.add("btn", "btn-default");
+    button.innerHTML = text;
+    button.onclick = callback;
+    button.style.marginRight = "4px";
+    return button
+}
+
+/**
+ * Creates a preset button {@see createPresetButton}
```

Comment:
```suggestion
 * Creates a link button
```

This function creates a button that leads to a link (href), as opposed to a preset. The description is misleading here.